### PR TITLE
Rock paper scissors pygame

### PR DESCRIPTION
**Pull request by Kyle-Sang**

### CODE REVIEW

**Comment by Kyle-Sang** [comment]
https://user-images.githubusercontent.com/78330032/122707593-4e5a5600-d228-11eb-81cd-ec25f9619d65.mp4

**Comment by Kyle-Sang** [comment]
There isn't a direct way to write to a Surface object, so the font is only available in human mode of the render.

**Comment by Kyle-Sang** [comment]
RPS_Test2

File: pettingzoo/classic/rps/rps.py [hunk 1]
```diff
@@ -63,12 +75,101 @@ def reinit(self):
...
+        lizard = pygame.transform.scale(lizard, (int(screen_height / 9 * (9 / 18)), int(screen_height / 9)))
+
+        # Set background color
+        bg = (220, 220, 220)
+        self.screen.fill(bg)
+
+        if mode == "human":
+            # Set font properties
+            black = (0, 0, 0)
+            font = pygame.font.Font('freesansbold.ttf', 32)
```
Comment (benblack769) [inline pettingzoo/classic/rps/rps.py:117]: The font should render in rgb_array mode as well. I belive you have to call `pygame.font.init()` to make fonts work.

**Comment by Kyle-Sang** [comment]
https://user-images.githubusercontent.com/78330032/123334692-76d89d80-d511-11eb-9582-f71e56a4870d.mp4

**Comment by justinkterry** [comment]
This is a big improvement.

-Please center the titles
-Please make the white space around all sides be the same as the small amount on the top
-Please make everything smaller still. These will ultimately be part of like 1 inch tall gifs on a website.

**Comment by Kyle-Sang** [comment]
https://user-images.githubusercontent.com/78330032/124038911-efd36b80-d9cf-11eb-9277-8dd6b2521d9a.mp4

**Pull request merged**